### PR TITLE
merge 8.2.x to main

### PR DESCRIPTION
**Pull request by dhalbert**
@UnexpectedMaker could you look over the revised files for your boards? Thanks.

### CODE REVIEW

**Review by tannewt — CHANGES_REQUESTED** [review]
I think you want the flash to be `qio` as well.

File: ports/espressif/boards/unexpectedmaker_bling/mpconfigboard.mk [hunk 1]
```diff
@@ -0,0 +1,22 @@
+USB_VID = 0x303A
+USB_PID = 0x8180
+USB_PRODUCT = "BLING!"
+USB_MANUFACTURER = "UnexpectedMaker"
+
+IDF_TARGET = esp32s3
+
+CIRCUITPY_ESP_FLASH_SIZE = 8MB
+CIRCUITPY_ESP_FLASH_MODE = dio
```
[the remark this answers is not in the document]
Comment (dhalbert) [reply ports/espressif/boards/unexpectedmaker_bling/mpconfigboard.mk:9]: confirmed by @UnexpectedMaker

File: ports/espressif/boards/unexpectedmaker_tinywatch_s3/mpconfigboard.mk [hunk 2]
```diff
@@ -0,0 +1,21 @@
+USB_VID = 0x303A
+USB_PID = 0x81B1
+USB_PRODUCT = "TinyWATCH S3"
+USB_MANUFACTURER = "UnexpectedMaker"
+
+IDF_TARGET = esp32s3
+
+CIRCUITPY_ESP_FLASH_SIZE = 8MB
+CIRCUITPY_ESP_FLASH_MODE = dio
```
[the remark this answers is not in the document]
Comment (dhalbert) [reply ports/espressif/boards/unexpectedmaker_tinywatch_s3/mpconfigboard.mk:9]: confirmed by @UnexpectedMaker

**Review by tannewt — APPROVED** [review]
Thank you!

**Comment by UnexpectedMaker** [comment]
Thanks @dhalbert !

**Pull request merged**